spiders/SheinData.py

```diff
@@ -0,0 +1,57 @@
+import requests
+from bs4 import BeautifulSoup
+import json
+import time
+
+http = 'https://www.shein.com/sitemap-index.xml'
+r = requests.get(http)
+soup = BeautifulSoup(r.text, 'xml')
+
+site_map_link = [] 
+linkprod = []
+Product_link = []
+category = []
+useless_links = []
+links = soup.find_all('loc')
+
+for ele in links:                           # Filtrage links Products o lcategories o useless links
+    if 'products' in ele.text:
+         site_map_link.append(ele.text)
+    elif 'category' in ele.text:
+            category.append(ele.text)
+    else:
+         useless_links.append(ele.text)
+
+i=0
+for collection in site_map_link:                   # Get Product links from link
+    print('_____________________________________________________________________________')
+    print(collection)
+    print('_____________________________________________________________________________')
+    rs = requests.get(collection)
+    soup_collection = BeautifulSoup(rs.text,'xml')
+    collection_items = soup_collection.findAll('loc')
+    for col in collection_items:
```

Comment:
you may use `for i, col in enumerate(collection_items)` it is the same as initializing i and keeps adding 1 to it